src/main/resources/db/migration/V20230711_26__IJPS-26_johPayroll.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE IF EXISTS joh_payroll
+  DROP CONSTRAINT IF EXISTS pk_joh_payroll;
+ALTER TABLE IF EXISTS joh_payroll
+  DROP CONSTRAINT IF EXISTS fk_johp_judicial_office_holder;
+DROP TABLE IF EXISTS joh_payroll;
+
+CREATE TABLE joh_payroll (
+    joh_payroll_id bigserial constraint pk_joh_payroll primary key,
+    local_joh_record_id bigint NOT NULL,
+    effective_start_date timestamp not null,
```

Comment:
As per the ticket its date instead of timestamp.

Reply:
done